Dockerfile

```diff
@@ -1,16 +1,46 @@
-FROM 		phusion/baseimage
-MAINTAINER 	Casey Bisson <casey.bisson@gmail.com
+FROM alpine:3.2
+MAINTAINER 	Casey Bisson <casey.bisson@gmail.com>
 
-# Update Apt
-RUN apt-get update
+# Alpine packages
+RUN apk --update \
+    add \
+        jq \
+        curl \
+        bash \
+        ca-certificates && \
+    curl -Ls https://circle-artifacts.com/gh/andyshinn/alpine-pkg-glibc/6/artifacts/0/home/ubuntu/alpine-pkg-glibc/packages/x86_64/glibc-2.21-r2.apk > /tmp/glibc-2.21-r2.apk && \
```

Comment:
I see it -- we're not building consul our selves and instead we're using a build we're getting from "mitchelh", which was built against libc:

```
# ldd consul
        linux-vdso.so.1 (0x00007fffffd9c000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f535d222000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f535ce79000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f535d43f000)
```

(Go binaries aren't always entirely statically linked)